rpm/packetfence.spec

```diff
@@ -1,87 +1,36 @@
-Vendor: PacketFence, http://www.packetfence.org
-
-# if --define 'snapshot 1' not written when calling rpmbuild then we assume it is to package a release
-%define is_release %{?snapshot:0}%{!?snapshot:1}
-%if %{is_release}
-# used for official releases
-Source: http://www.packetfence.org/downloads/PacketFence/src/%{real_name}-%{version}.tar.gz
-%else
-# used for snapshot releases
-Source: http://www.packetfence.org/downloads/PacketFence/src/%{real_name}-%{version}-%{rev}.tar.gz
-%endif
-
-# Log related globals
-%global logfiles packetfence.log snmptrapd.log pfdetect pfmon security_event.log httpd.admin.audit.log
-%global logdir /usr/local/pf/logs
+#==============================================================================
+# Variables
+#==============================================================================
+# use 'global' variables (vs 'define' with local scope)
+%global     builddoc 0
+%global     perl_version 5.10.1
+%global     logfiles packetfence.log snmptrapd.log pfdetect pfmon security_event.log httpd.admin.audit.log
+%global     logdir /usr/local/pf/logs
+%global     debug_package %{nil}
+
+
+#==============================================================================
+# Main package
+#==============================================================================
+Name:       packetfence
+Version:    9.1.9
+Release:    1%{?dist}
+Summary:    PacketFence network registration / worm mitigation system
+Packager:   Inverse inc. <support@inverse.ca>
+Group:      System Environment/Daemons
+License:    GPL
+URL:        http://www.packetfence.org
+Source0:    %{name}-%{version}.tar
+BuildRoot:  %{_tmppath}/%{name}-root
+Vendor:     PacketFence, http://www.packetfence.org
 
 BuildRequires: gettext, httpd, ipset-devel, pkgconfig, jq
-# Required to build documentation
-# See docs/docbook/README.asciidoc for more info about installing requirements.
-# TODO fop on EL5 is actually xmlgraphics-fop
 BuildRequires: asciidoc >= 8.6.2, fop, libxslt, docbook-style-xsl, xalan-j2
-BuildRequires: asciidoctor
+BuildRequires: ruby, rubygems
+BuildRequires: nodejs >= 12.0
 BuildRequires: gcc
+BuildRequires: systemd
 
-%description
-
-PacketFence is an open source network access control (NAC) system.
-It can be used to effectively secure networks, from small to very large
-heterogeneous networks. PacketFence provides features such as
-* registration of new network devices
-* detection of abnormal network activities
-* isolation of problematic devices
-* remediation through a captive portal
-* registration-based and scheduled vulnerability scans.
-
-# arch-specific pfcmd-suid subpackage required us to move all of PacketFence
-# into a noarch subpackage and have the top level package virtual.
-%package -n %{real_name}
-Group: System Environment/Daemons
-Summary: PacketFence network registration / worm mitigation system
-BuildArch: noarch
-# TODO we might consider re-enabling this to simplify our SPEC
 AutoReqProv: 0
 
```

Comment:
You will need to play with : Obsoletes: packetfence--remote-arp-sensor, packetfence-...

Reply:
Ok, I will look into this.

Reply:
Done.

Reply:
As I said yesterday, I will try to upgrade from 9.1.0 packages to 9.1.9 packages to see if it works without conflicts.